### DISCUSSION TITLE
Missng Sources and Medium data

### DISCUSSION THREAD

**Discussion by krzyzanowskim** [plausible/analytics#935] [Anything Else]
Hi, I've been using Plausible for some time, and I noticed I never get anything in Sources, Medium and I wonder what's going on.

My integration is with a script, I also did connect Google Console but can't find any data.

```
<script async defer data-domain="domain.app" src="https://plausible.io/js/plausible.js"></script>
```

![Screen Shot 2021-04-16 at 13 31 50](https://user-images.githubusercontent.com/758033/115018533-41338a80-9eb8-11eb-9e84-c3c097d5a4f2.png)

what I may be missing here?

**Comment by metmarkosaric** [comment]
thanks @krzyzanowskim! are you using UTM tags? medium/source/campaign is where traffic tagged with UTM tags goes https://plausible.io/docs/top-referrers

**Reply by krzyzanowskim** [reply]
I don't use UTM. You're saying it's only from UTM, and not trying to do it otherwise? Ok, that was not my assumption, and I expected to see at least something in theger.
